src/rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    @book.rental = self
+    @person.rental = self
```

Comment:
- [ ] Kindly notice that the `book` and `person` has a `rentals` attribute and it is an array, so you should add the rental to it not assign it to it.

```ruby
book.rentals << self
person.rentals << self
```